requirements/base.in

```diff
@@ -40,10 +40,7 @@ git+https://github.com/maykinmedia/python3-saml.git@58fd3ad798510c7ced110040d2fd
 # once transferred from isprojects to maykinmedia, we'll publish this to PyPI
 git+https://github.com/isprojects/drf_jsonschema.git@b7a482670274d11c1849e625a704018f1e9dd852#egg=drf_jsonschema
 pyopenssl
-# WIP maykin-django-two-factor-auth[phonenumbers]
-# This commit is a commit on the branch task-334-fix-login-page-style, which is still undergoing some
-# tweaks
-git+https://github.com/maykinmedia/django-two-factor-auth.git@549bef1586eaa9088058352889e0747889090bba#egg=maykin-django-two-factor-auth[phonenumbers]
+maykin-django-two-factor-auth[phonenumbers]==2.0.3
```

Comment:
Can you remove the pinned version number from the `base.in` file? You can always feed the `-P` option to `./bin/compile_dependencies.sh` which achieves the same result, e.g.:

```bash
./bin/compile_dependencies.sh -P maykin-django-two-factor-auth[phonenumbers]==2.0.3
```

without pinned version, it will try to update the package to the latest available version (within the constraints of course)

Reply:
Ah, my bad.